tests: windows sqlite test is broken due to spaces in the version number

* related: #2497 

It looks like @ffontaine 's changes in #2497 has resulted in test breakage on windows:

```python
FAILED test/test_scanner.py::TestScanner::test_version_mapping[sqlite-2020.04.06 18:16:31 1e4b6a93987cdfbf829e2ff35ef417c290625f2894ad11949e301af518f1fb44-version_strings323] - OSError: [Errno 22] Invalid argument: 'C:\\Users\\RUNNER~1\\AppData\\Local\\Temp\\mapping-test-41t_l8qa\\tmp9lbgvwpo-sqlite-2020.04.06 18:16:31 1e4b6a93987cdfbf829e2ff35ef417c290625f2894ad11949e301af518f1fb44.out'
```

My bad for not noticing this before merging.  It looks like it's because the version now has a space in it so the temp filename isn't working as planned.  The fix won't be in the checker, it'll likely need to be in the test files for this one so we convert spaces to something appropriate when making test files.

--- a/test/test_scanner.py
+++ b/test/test_scanner.py
@@ -125,7 +125,7 @@ class TestScanner:
             # Replace space in filename to avoid a failure on Windows
             with tempfile.NamedTemporaryFile(
                 "w+b",
-                suffix=filename.replace(" ", "_"),
+                suffix=filename.replace(":", "_"),
                 dir=self.mapping_test_dir,
                 delete=False,
             ) as f:

--- a/test/test_scanner.py
+++ b/test/test_scanner.py
@@ -122,8 +122,12 @@ class TestScanner:
             f"{'.'.join(list(product))}-{version}.out",
         ]
         for filename in filenames:
+            # Replace space in filename to avoid a failure on Windows
             with tempfile.NamedTemporaryFile(
-                "w+b", suffix=filename, dir=self.mapping_test_dir, delete=False
+                "w+b",
+                suffix=filename.replace(" ", "_"),
+                dir=self.mapping_test_dir,
+                delete=False,
             ) as f:
                 f.write(b"\x7f\x45\x4c\x46\x02\x01\x01\x03\n")
                 f.writelines(version_strings)

